src/components/templates/ConversationSpace/ConversationSpace.tsx

```diff
@@ -24,14 +25,15 @@ export const ConversationSpace: React.FunctionComponent = () => {
   const venue = useSelector(currentVenueSelectorData);
   const { recentVenueUsers } = useRecentVenueUsers();
 
+  const defaultTables = venue?.config?.tables ?? TABLES;
+  const [tables, setTables] = useState(defaultTables);
```

Comment:
`useState(defaultTables);` This default value will be set only once on the initial render, is this what we want? Are you sure that this won't create a problem later on.

Reply:
IMO I should store `defaultTables` and toggle them by filtering (if they are locked or full). I assume yes, it's ok to set them on the initial render.
Please clarify what problem it could create later?

Reply:
Well, for instance, if the prop updates and `defaultTables` changes, this won't re-trigger itself and the default values will be the same.

Reply:
Well, I've fixed the edge case when the `defaultTables` are changed in 876c4407b981917b01639e7768d6a3a9d6ef75e8